python/cuspatial/cuspatial/core/_column/geocolumn.py

```diff
@@ -179,6 +179,25 @@ def copy(self, deep=True):
         )
         return result
 
+    @property
+    def valid_count(self) -> int:
+        """
+        Returns the number of valid geometries stored in this GeoColumn.
+        Null support is implemented in GeoSeries and this is not expected
+        to ever return less than the length of the GeoColumn. This is
```

Comment:
Do we really want to document this expectation in this way? Might cause a user to make assumptions or rely on this expectation.

Reply:
I modified the docs, take another look?